### ISSUE TITLE
Project Plan

### ISSUE THREAD

**Issue by lromualdo24** [stat231-f21/blog_revisiting-2020#1]
For our blog project, we will continue to build on the visualizations we made for the Shiny App project, with the central theme question of "What factors influenced the 2020 election?" Our plan is as follows:

1. Use the interactive state map as a 'cover page' of our blog to provide an overview of the election for readers
2. Use the interactive scatterplot, correlation plot, and wordcloud to provide examples of factors (geographic, economic, public health, etc) that influenced the election

In order to build on our past work, we will develop our current visualizations in the following ways:

1. Creating new wordclouds by candidate, looking for mentions of each 2020 candidate in articles and finding what words appear near the mention and what sentiment they have, on average
2. Wordcloud clustering, perhaps by issue or candidate
3. Adding a correlation plot between region (and perhaps ones with more states) to see how different regions of the country voted in relation to each other, not just states within the regions
4. Update the datasets in the interactive scatterplot, including corrections to COVID-19 data and adding stock performance of companies tied to each political party, to see how the stocks of politically-connected firms varied with candidates' performance in the polls

Our plan to accomplish this work is as follows:
1. Collect new data needed for added components by 11/9
2. Finish minor updates to visualizations (dataset updates for scatterplot, updated correlation plot) by 11/16
3. Complete Wordcloud additions, have a rough first draft or out line for blog by 11/30
4. Finish blog by 12/03

We will have weekly check-ins, with more or less as needed depending on the team's progress.

**Comment by bebailey** [comment]
This was not part of the requirement, but please make sure you discuss amongst yourselves exactly how you plan to divide up the work. Perhaps it would be useful to write it down as a sort of group contract, even. To the degree possible, we want equitable contributions to the core components of the blog, with flexibility for any individual to go "above and beyond" on any particular aspect of interest to them. 

---

To embed a shiny app, you can use a code chunk (see [this link](https://bookdown.org/yihui/bookdown/web-pages-and-shiny-apps.html) for details):

```
knitr::include_app("https://your-account.shinyapps.io/your-app/", height = "600px")
```

**Comment by pbakshi24** [comment]
Status Update #1:
We have a considerable amount of progress on our blog project. For our scatterplot, the COVID datasets were updated with data collected after we downloaded them for the Shiny presentation and we uploaded new stock datasets. In regards to the outline for the blog, we made the RMD outline and started embedding the shiny apps. For the features of the app, we are in the process of adding a heat map showing the differences in urbanization between each state to see how this could potentially effect election outcomes. We were originally going to add a correlation heat map of the regions of the US but realized that showing the entire heat map wasn't very informational or useful for the purpose of the blog. Lastly, we are still working for R Selenium to scrape information from dynamically loaded web pages. It seems like the Wall Street Journal has different loading mechanisms for their content pages, so this is taking more time to figure out than we expected. We are overall still on schedule for how much we wanted to complete by this point.

**Comment by pbakshi24** [comment]
Status Update #2:
We have continued making progress on our project! One thing that has changed from our original plan is one of our visualizations. Rather than adding a heat map showing the differences in urbanization by state, we decided that it would be more informative and clear to add a scatterplot showing the relationship between urbanization and the margin that each candidate was predicted to win by. We are slightly ahead of schedule as we have already created the outline for our blog and are almost completely done with visualizations. We are still finalizing Word cloud additions but are looking to be done with that soon as well.

**Comment by bebailey** [comment]
I'm a little concerned that no one has pushed any commits to the repo since I created it... Are you working in a different repo? Please make sure to push all relevant files here (sooner rather than later so you can troubleshoot any conflicts).

**Comment by lromualdo24** [comment]
Because our project is an extension of our SHINY app work, we are continuing to work in the same repo as that app until each of the visualizations are complete (we will then transfer those files to this repo once they are no longer a work in progress). As for the blog Rmd itself, we have been working out of a text editor to complete the written portion more efficiently, and will integrate it into a markdown file once the bulk of the text + analysis is complete. Let us know if this setup is OK.
